tests/impls/gen_detector_prop_sal/test_drise_scoring.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import pytest
+import os
+import re
+
+from xaitk_saliency.impls.gen_detector_prop_sal.drise_scoring import DRISEScoring
+
+from xaitk_saliency import GenerateDetectorProposalSaliency
+from smqtk_core.configuration import configuration_test_helper
+from tests import DATA_DIR, EXPECTED_MASKS_4x6
+
+
+class TestDRISEScoring:
+
+    def test_init_(self) -> None:
+        """
+        Test if implementation is usable.
+        """
+        impl = DRISEScoring()
+        assert impl.is_usable() and isinstance(impl, GenerateDetectorProposalSaliency)
+
+    def test_default_param(self) -> None:
+        """
+        Test default construction.
+        """
+        impl = DRISEScoring()
+        assert impl.proximity_metric == 'cosine'
+
+    def test_get_config(self) -> None:
+        """
+        Test expected configuation behavior.
+        """
+        impl = DRISEScoring('euclidean')
+        for i in configuration_test_helper(impl):
+            assert i.proximity_metric == 'euclidean'
+
+    def test_metric_args(self) -> None:
+        """
+        Test non-default metric type.
+        """
+        impl = DRISEScoring('hamming')
+        assert impl.proximity_metric == 'hamming'
+
+    def test_bad_metric_args(self) -> None:
+        """
+        Test non-default metric type.
+        """
+        with pytest.raises(
+            ValueError,
+            match=r"Chosen comparision metric not supported or "
```

Comment:
```suggestion
            match=r"Chosen comparison metric not supported or "
```

Reply:
Thanks for catching this typo! I've fixed this and the other instances below.